experimental/devices/mcp9808/mcp9808.go

```diff
@@ -264,10 +264,10 @@ func (d *Dev) readTemperature() (physic.Temperature, uint8, error) {
 		return 0, 0, errReadTemperature
 	}
 	// Convert to physic.Temperature 0.0625°C per bit
-	t := physic.Temperature(tbits&0x0FFF) * 62500 * physic.MicroKelvin
+	t := physic.Temperature(tbits&0x0fff) * 62500 * physic.MicroKelvin
```

Comment:
I don't think I've been consistent on the casing of hex values, but I'd prefer the ones that do not have the value changed to not be edited, so the blame list is not affected.

Reply:
Fair enough. Reverted.